collectors/ebpf.plugin/README.md

```diff
@@ -244,19 +244,29 @@ To collect data related to Linux multi-device (MD) flushing, the following kprob
 
 ### Disk
 
-The eBPF plugin also shows a chart in the Disk section when the `disk` thread is enabled. This will create the
-chart `disk_latency_io` for each disk on the host. The following tracepoints are used:
+The eBPF plugin also shows a chart in the Disk section when the `disk` thread is enabled.
+
+#### Disk Latency
+This will create the chart `disk_latency_io` for each disk on the host. The following tracepoints are used:
 
 - [`block/block_rq_issue`](https://www.kernel.org/doc/html/latest/core-api/tracepoint.html#c.trace_block_rq_issue):
   IO request operation to a device drive.
 - [`block/block_rq_complete`](https://www.kernel.org/doc/html/latest/core-api/tracepoint.html#c.trace_block_rq_complete):
   IO operation completed by device.
 
+Disk Latency is the single most important metric to focus on when it comes to storage performance, under most circumstances.
+For hard drives, an average latency somewhere between 10 to 20 ms can be considered acceptable. For SSD (Solid State Drives),
+depending on the workload it should never reach higher than 1-3 ms. In most cases, workloads will experience less than
```

Comment:
```suggestion
depending on the workload it should never reach higher than 1-3 ms. In most cases, workloads will experience less than
```

I'm going to suggest we combine this sentence and the next one to say "In most cases, workloads experience less than 1 ms latency numbers, but workloads should never reach higher than 3 ms."